app/models/EpicTest.scala

```diff
@@ -33,7 +33,7 @@ case class EpicVariant(
   separateArticleCount: Option[SeparateArticleCount],
   showChoiceCards: Option[Boolean],
   bylineWithImage: Option[BylineWithImage],
-  defaultChoiceCardFrequency: Option[ContributionType],
+  defaultChoiceCardFrequency: Option[String],
```

Comment:
this can still have a type, `ContributionType`

Reply:
I deleted the old `app/models/ContributionTypes.scala` file containing the `ContributionType` definition as the contribution types page in RRCP has been deleted (as agreed). 

What I haven't done is remove this optional field from the EpicVariant definition, as all existing Epic variants still include this data even though the ability to set which contribution type (one-off, monthly, annual) should display as the default is (has long been) ignored by amounts cards in the Epic - which instead will use the amounts test variant equivalent data to set the default tab.

I have removed the ability for users to indicate which contribution amount to display when creating an epic variant as it was both redundant, and confusing UX.

Reply:
Definitely good for removal in a future PR